charts/camunda-platform-alpha/templates/identity/configmap.yaml

```diff
@@ -30,20 +35,25 @@ data:
         issuer-url: {{ include "camundaPlatform.authIssuerUrl" . | quote }}
         backend-url: {{ include "camundaPlatform.authIssuerBackendUrl" . | quote }}
 
+      {{- if ne .Values.global.identity.auth.type "KEYCLOAK"}}
+      initial-claim-name: {{ .Values.global.identity.auth.identity.initialClaimName }}
+      initial-claim-value: {{ .Values.global.identity.auth.identity.initialClaimValue }}
+      {{- end }}
```

Comment:
initialClaimName + initialClaimValue look like OIDC settings.  In azure, they determine what attribute of a users account would determine whether they are the admin user.  But I don't think these options are relevant for keycloak.

Reply:
that's correct, that's why the if condition here is `!= KEYCLOAK`.

Reply:
that's correct, that's why the if condition here is `!= KEYCLOAK`.